mslib/msui/mscolab.py

```diff
@@ -259,22 +279,38 @@ def text_changed(self):
         self.loginButton.setEnabled(self.emailid.text() != "" and self.password.text() != "")
 
     def handle_import(self):
-        file_path = get_open_filename(self, "Select a file", "", "Flight track (*.ftml)")
+        if self.workLocallyCheckBox.isChecked():
```

Comment:
Why not here...?

Reply:
if there is  request we can add this. Also there is a webui project.